spec/koans.js

```diff
@@ -221,78 +225,78 @@ describe('The object literal allows for new shorthands. ', () => {
 describe('destructuring arrays makes shorter code. ', () => {
 
   it('extract value from array, e.g. extract 0 into x like so `let [x] = [0];`', () => {
-    let firstValue = [1];
-    //expect(firstValue).toEqual(1);
+    let [firstValue] = [1];
+    expect(firstValue).toEqual(1);
   });
 
   it('swap two variables, in one operation', () => {
     let [x, y] = ['ax', 'why'];
-    [x, y] = [x, y];
-    //expect([x, y]).toEqual(['why', 'ax']);
+    [y,x] = [x, y];
+    expect([x, y]).toEqual(['why', 'ax']);
   });
 
   it('leading commas', () => {
     const all = ['ax', 'why', 'zet'];
-    const [z] = all;
-    //expect(z).toEqual('zet');
+    const z = all[2];
```

Comment:
The shortcut for accessing a special element in an array is like that : 
```js
  const [,,z] = all;
```
Since you want to put the third item of the array in a variable you call z, you have to add two commas before. This seems a bit odd and more complicated than the normal way at first, but you'll see useful use cases of this if you do many Katas on codewars :)